kustomize/overlays/stable/secrets/keycloak-secrets.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  annotations:
+    sealedsecrets.bitnami.com/cluster-wide: "true"
+  creationTimestamp: null
+  name: keycloak-secrets
+spec:
+  encryptedData:
+    KEYCLOAK_PASSWORD: AgAI23aRUVsp7YZnoys7Ij9RGV0KZQOSR0YqG+GVFmbOQSgKPZyBnmuL3SBeTM3eVHqg/6QFsqZSVXQsgry/3q2uj+eX80zXlO5N7MQk1LaY1wadau/GgCN87TbK8UShilRa7YIq2Ow73V/93jaVAt4TJkYrz3UryVcwRdWzE8at3dbi4z3JFsbof8p9x2nLggSkFijHuqW73UfFk+vKtlu78Ek7QlzkgcqJkUK5IhDmdxKUx6uB5Zapim2MAJRsTWqVWbKmZnRj8LIhq1Bk6WLsn+U87pUP6kQjwuYTNObMaRUNeXNXqgT9faIw4aYOWXqyGSOUg9PoXfhm2mbddT4GXzxw/wqf6wOpjESaz2njDiJqfsW2k5LBTA8Y0XqSWfPdoFgC/5ahHavDXnIzOdE6fTxxBtRVYqZBaPVG0ywjj6UOqmUctfm6ZOBXkcFHwoT34XL/FJ8LbjLDn2JBgRZK4nv1kxXRuoP3w+3FzBtiu5EHIN7Emwi4mjZ5blI+p1bBhyGkOMswCQN1Obydifvw0j5lyU6Bl47/acRDpNtVErPt7bY/e5Ac/vXGbOBdyPKMn40/40ffffuNRykkveEcAVHluRZ15G3uKoqywaGRWIdSNU92VdmiTvH52BUHvvNujNzeqN/4Up4SbT7vld0BnthsqoQCWu9I8Vv88Rn14MWIWeQhjDRLfnodFOhYj0mc/pV1V9NK06uXAz1zkLM=
```

Comment:
@kostola I assume the key used to encrypt these values is specific to our stable cluster?